content/docs/tasks/security/health-check.md

```diff
@@ -41,7 +42,7 @@ this feature is not needed if the production setup is not using the
 Deploy Citadel with health checking enabled.
 
 ```command
-$ kubectl apply -f @install/kubernetes/istio-citadel-with-health-check.yaml@
+$ kubectl apply -f install/kubernetes/istio-citadel-with-health-check.yaml
```

Comment:
Please add back the @@ symbols. Those render as links to that file on GitHub. See https://istio.io/about/contribute/writing-a-new-topic#showing-references-to-istio-github-files

Reply:
@geeknoid those files are not part of the distribution.